index.d.ts

```diff
@@ -75,6 +75,10 @@ export function combineReducers<S>(reducers: ReducersMapObject): Reducer<S>;
 
 /* store */
 
+export interface MiddlewareDispatch {
+    <TMiddlewareAction, TMiddlewareActionResult>(action: TMiddlewareAction): TMiddlewareActionResult;
```

Comment:
it will not infer types automatically. can you define the function that will not require providing types explicitly?
If I call it without types, TS will infer `{}` as a `TMiddlewareAction`.